documentation/modules/auxiliary/dos/windows/http/rb_dos_iis_2021_31166.md

```diff
@@ -0,0 +1,61 @@
+## Vulnerable Application
+
+IIS web server on Windows 10 and Windows Server version 20H2 and 2004 without KB5003173.
+
+## Options
+
+### RHOST
+
+ - **Required**
+ - Type: **address**
+ - *No default value*
+
+IP address or hostname of the target IIS server.
+
+### RPORT
+
+ - **Required**
+ - Type: **integer**
+ - Default value: **80**
+
+The port on the target server where IIS is running.
+
+### TARGETURI
+
+ - **Optional**
+ - Type: **string**
+ - Default value: **/**
+
+The base URL of the IIS install on the target server.
+
+## Scenarios
+
+```text

```

Comment:
```suggestion
### Windows XXX running IIS XXX with <MONTH> <YEAR> Patches
```text

```

Reply:
I will add the configuration of my Virtual Machine.

Reply:
Do you want me to add the version and patch like this: 

### Windows 10 20H2 19042.804 running IIS with November 22, 2021 Patches (KB5007253)